src/roswire/common/format.py

```diff
@@ -56,57 +63,63 @@ def build(db: PackageDatabase) -> "FormatDatabase":
                 if action.feedback:
                     messages.add(action.feedback)
 
-        return FormatDatabase(messages, services, actions)
+        return cls(messages, services, actions)
```

Comment:
We probably want to either: (a) change this to `cls.build(...)`, or (b) remove `build` entirely since it's really just an alias for the constructor.

Reply:
Good point. I have removed the build method